pkg/utils/util.go

```diff
@@ -586,3 +593,40 @@ func IsGroupSnapshotCreated(groupSnapshot *crdv1alpha1.VolumeGroupSnapshot) bool
 func GetDynamicSnapshotContentNameForGroupSnapshot(groupSnapshot *crdv1alpha1.VolumeGroupSnapshot) string {
 	return "groupsnapcontent-" + string(groupSnapshot.UID)
 }
+
+// ShouldEnqueueContentChange indicated whether or not a change to a SnapshotContent object
```

Comment:
nit: SnapshotContent -> VolumeSnapshotContent

Can you also add some explanation here, i.e., what changes should be ignored, similar to what you have added to the PR description.